### PR TITLE
docs(mdbook): Moved Angular documentation from ClickUp to mdbook.

### PR DESCRIPTION
**Pull request by Dom-Roth**
Greetings, this update extends the new mdbook by the Angular documentation.

There are some links to https://composablefinance.atlassian.net/ which isn't active anymore.
Does anymore have the files/docs hosted there, so I can move them here?

### CODE REVIEW

**Review by dzmitry-lahoda — APPROVED** [review]

**Comment by cor** [comment]
This PR looks good to me, I propose that we merge it after we merge the `cor/mdbook-migration` (later today) so that we have a good test to see if merging this PR correctly deploys the updated book

**Review by dzmitry-lahoda — DISMISSED** [review]

File: book/src/products/angular/technical-specification/liquidations.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
...ank&layers=1&nav=1&title=dutch-auction-pl.drawio#R7Vlbb9owFP41eezk3Lg8tsDaaZ2G1Enr%2BmaS08RSEiPHgdBfP4c4JLahNwFh0hAP9ufjS75zzucTsNxJWt4yvIx%2F0BASy0FhablTy3Fsz3Gs6ovCjUTGtkQiRkKJtcADeQEJIokWJIRcMeSUJpwsVTCgWQYBVzDMGF2rZs80UXdd4ggM4CHAiYn%2BJiGPa3Tkoxa%2FAxLFzc42kiMpbowlkMc4pOsO5M4sd8Io5XUrLSeQVOw1vNTzvh4Y3R2MQcbfMwHT1eqq8O%2Fjpzz%2FvvjGnvJn90qussJJIR9YHpZvGgYgC68rIkUvSHCek8Byb2KeJgKwRZPRIguh2gSJHoSCPjmZMh7TiGY4mbXojXlueYhqageQT3ELNAXONsKAQYI5WamewdLB0c5uN3VOidjCQTIaPekIGYojpC6Q04IFIOd0eXx9Gc%2FT1uGYRcCNdUSj8zAttPXSBzzmGB77RVIwvSbIvMcLkY7CQ5CTF7zYDiHVeTghUVZ5VjgDmABWwDgR4X8tB1IShgfc9lpAyXSUm7ZJUC0P5XG8rDD7URp9g0bb4FCN7HVMODwscVCNroXcqUweDOt3PHKpRqQWoOtWdwYSijuSM0AnomhgUGSKQ88U2YOeORoaHLmXxtFOsPriaGRw5F0aR67TM0djgyP%2F0jjyUM8cNSVhh6TrgFNmECUqrWXVLNKkNmivte2VOKc54YRW19uCck7TPfcepxqhtOAJyWCyqzPRcVh2bS1dbZNm96w0uwbNBsPHrgmhJPyxsRbtP1X7i6t95OC07FhON53OHJiohbaFTI1lgpBHeaZtp17Yb7rtUtve5i231jXiO2K0LgLfvlxPXfP6anCNP1nzast4%2FnlrXtszQvLnIgcmaHHQnJHggupfmT0XXwDbZgU8nT32ful4eoW352YenlUOzTL4bHJon0gO0cXJYRONp9bDXaGnF8cfVURbk0QHnVkSzTePWQlBwStJvDPfZXvTw8G%2Foofma8r%2FsufoeT48U5rr2fnZNPfG2nWk3zMH0lyECd50zJaVQX74wI62T1PyHzqXYT9U7EWjPsFxNcd8S%2B1ozp506UtzRr1rjui2v%2BzX%2FLd%2FkLizvw%3D%3D"></iframe>*
```
[the remark this answers is not in the document]
Comment (Dom-Roth) [reply book/src/products/angular/technical-specification/liquidations.md:35]: As was I! :joy:

[the remark this answers is not in the document]
Comment (Dom-Roth) [reply book/src/products/angular/technical-specification/liquidations.md:35]: Funnily for some unknown i had to wrap it inside *  * characters, else it wouldn't work.

**Review by cor — APPROVED** [review]
LGTM

**Review by hussein-aitlahcen — APPROVED** [review]

**Pull request merged**